pkg/studyengine/expressions.go

```diff
@@ -1175,6 +1183,44 @@ func (ctx EvalContext) not(exp types.Expression) (val bool, err error) {
 	return
 }
 
+// sum(...float64) each argument must resolve to a float value or will be disguarded
+func (ctx EvalContext) sum(exp types.Expression) (t float64, err error) {
+	for idx, dataExp := range exp.Data {
+		arg, err := ctx.expressionArgResolver(dataExp)
+		if err != nil {
+			logger.Debug.Printf("exp %d in 'sum' returned error: %v", idx, err)
+			continue
+		}
+		switch v := arg.(type) {
+			case bool:
+				if(v) {
+					t = t + 1
+				}
+			case float64:
+				t = t + v
```

Comment:
@cturbelin looks good to me so far, maybe one more debug print here, for the default switch case (with unexpected type at index)?